flake-info/README.md

```diff
@@ -68,7 +68,8 @@ The `<flake>` argument should contain a valid reference to a flake. It accepts a
 
 > use git+<url> to checkout a git repository at <url>
 > use /local/absolute/path or ./relative/path to load a local source
-> use gitlab:<user>/<repo>/github:<user>/<repo> to shortcut gitlab or github repositories
+> use gitlab:<user>/<repo>/github:<user>/<repo>/sourcehut:<user>/<repo> to
+> shortcut gitlab, github or sourcehut repositories
```

Comment:
Could you remove the entire quotation block here and add a link to https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake.html#types instead?

Reply:
Ah, but not all types supported by flakes are supported by `nix-search`, are they?